src/importTest/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Dependency directories
```

Comment:
I think we should likely put this inside of a test directory still. That will make it more clear that it is test related.

That way this will be `src/test/importTest` or perhaps just `test/importTest`.

Reply:
I also went ahead and put the test files into `src/test` along with the `importTest` directory